climada/util/plot.py

```diff
@@ -195,7 +195,7 @@ def _plot_scattered_data(method, array_sub, geo_coord, var_name, title,
     if axes is None:
         proj_plot = proj
         if isinstance(proj, ccrs.PlateCarree):
-            # use different projections for plot and data to shift the central lon in the plot
+            # for PlateCarree, center plot around data's central lon without overwriting the data's original projection info
             xmin, xmax = u_coord.lon_bounds(np.concatenate([c[:, 1] for c in list_coord]))
             proj_plot = ccrs.PlateCarree(central_longitude=0.5 * (xmin + xmax))
         _, axes, fontsize = make_map(num_im, proj=proj_plot, figsize=figsize,
```

Comment:
```suggestion
            proj = ccrs.PlateCarree(central_longitude=0.5 * (xmin + xmax))
        _, axes, fontsize = make_map(num_im, proj=proj, figsize=figsize,
```
Is the `proj_plot` variable necessary? Can't the `proj` just be replaced here in case it is PlateCarrree?

Reply:
The `proj_plot` is needed if the projection of the plot is different from the one of the data. The use case here is that one wants the plot to be centered around the data (so, eg., to avoid plotting the whole world when looking at the Fidji islands, which cross the date line).